### ISSUE TITLE
Describe SB Flavor Ontology

### ISSUE THREAD

**Issue by jag3773** [bible-technology/scripture-burrito#76]
We need to describe several different use cases and describe how they map on to our flavor and flavorType system.

**Comment by mvahowe** [comment]
I'm a little confused about this.

If the aim is to demonstrate that our system can describe multiple types of "thing", we've done that already. We have already described several different use cases and how they map onto our flavor and flavorType system.

If the aim is to demonstrate that our system can describe something like "everything anyone might want to describe", that's impossible to demonstrate empirically.

Beyond that, I think that https://docs.burrito.bible/en/latest/extending.html explains how to navigate the various options quite thoroughly. It would be nice if futurework could build on or critique what we have already, rather than starting another process in parallel.

In practice, the current system can represent pretty much anything, since it's a glorified directory structure plus metadata. I'm pretty sure I could spin up a "Paratext source code" flavor.

My concern with words like "ontology" is that I don't think there is a unique and agreed ontology for Bible-related content, and I don't think we're going to find one, because the issues are partly about fundamental theological categories. For example, does Dallas Theological Seminary consider Bel and the Dragon to be "Scripture"?

I see the flavorType/flavor/convention mechanism as pragmatic and, like any pragmatic solution, there are multiple ways to proceed. I fear we are going to be dragged into an endless discussion equivalent to "in exactly which use cases should programmers use inheritance rather than inclusion?" Really, the only way for SB to have any hope of being widely adopted is to stay out of this kind of conversation.

The way forward I had imagined, and which I still think we should pursue, is to actively invite the world to try representing things they actually want to represent, and see how that goes. We're trying that with versification, and I think we've demonstrated that none of the hard problems is with Scripture Burrito. The hard problems are about how different ecosystems use versification. It's not our job here to answer those questions... but, whatever the answers, if we ever find the answers, the chances are that we could represent those answers in SB.
